poi-ooxml/src/main/java/org/apache/poi/openxml4j/opc/PackagePartName.java

```diff
@@ -326,7 +326,7 @@ private static void checkPCharCompliance(String segment)
             /* Check rule M1.7 */
             if (decodedChar == '/' || decodedChar == '\\') {
                 throw new InvalidFormatException(
-                    "A segment shall not contain percent-encoded forward slash ('/'), or backward slash ('\') characters. [M1.7]");
+                        "A segment shall not contain percent-encoded forward slash ('/'), or backward slash ('') characters. [M1.7]");
```

Comment:
this change is wrong

Reply:
or at least - I think the aim is to produce the text `'\'` - the fix might be to use `'\\'`

Reply:
reverted.